.github/workflows/github-actions.yml

```diff
@@ -17,6 +17,6 @@ jobs:
       uses: actions/setup-node@v1
       with:
         node-version: ${{ matrix.node-version }}
-    - run: cp test/test_dot_env .env
+    - run: npm install -g npm@latest
```

Comment:
Is this going to impact installations from nvm?  I'd figure that the docker image would come packaged with the default version of npm, which would be the same as we'd get from nvm.

Wonder if we should just bump to node 14.

Reply:
I think I tried with node 15, but ran into dependency problems I wasn't able to sort out quickly, so I just went with keeping the same version of node but updating npm. It looks like nvm with node12 defaults to npm version 6, which is why this was needed.